lib/common/models/BlockchainLockModel.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Encapsulates data about locks created and retrieved by the blockchain service.
+ */
+export default interface BlockchainLockModel {
+  /** Uniquely identifies a lock */
+  identifier: string;
+
+  /** The amount that is locked */
+  amountLocked: number;
+
+  /** At this transaction time the lock is no longer valid */
+  lockEndTransactionTime: number;

```

Comment:
`lockEndTransactionTime: lockUntilBlock` in `LockResolver` is a bit unclear because `lockUntilBlock` seems to be inclusive. 

Consider `unlockTransactionTime` here and `unlockAtBlock` in `LockResolver`.